tests/test_datetime.py

```diff
@@ -95,12 +96,14 @@ def test_to_float_timestamp_absolute_no_reference(self):
         with pytest.raises(ValueError):
             tg_dtm.to_float_timestamp(dtm.datetime(2019, 11, 11), reference_timestamp=123)
 
-    @pytest.mark.parametrize("time_spec", DELTA_TIME_SPECS, ids=str)
-    def test_to_float_timestamp_delta(self, time_spec):
+    def test_to_float_timestamp_delta(self):
```

Comment:
why not parametrize?

Reply:
https://github.com/python-telegram-bot/python-telegram-bot/pull/3426/commits/3bdd98128a4f678c21f747b088df2ba0a2e9e21a

there's a tiny chance that there is an error while collecting the tests (happens when time goes from HH:59:00 -> HH+1:00:00)

This was happening before this PR as well

Reply:
Worth a CIS comment :)